openmdao/docs/openmdao_book/features/core_features/working_with_components/explicit_component.ipynb

```diff
@@ -133,8 +133,6 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "(matrix-free-api)=\n",
```

Comment:
looks like we still need the `(matrix-free-api)=` label..  it's linked [here](https://openmdao.org/newdocs/versions/latest/features/core_features/working_with_components/distributed_components.html#derivatives-with-distributed-variables)